.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,246 @@
+name: Build and upload to PyPI
+
+on:
+  pull_request:
+    branches:
+      - "package-*"
+  push:
+    branches:
+      - "package-*"
+    tags:
+      - "package-*"
+  release:
+    types:
+      - published
+
+
+concurrency:
+  group: "${{ github.ref }}-${{ github.head_ref }}-${{ github.workflow }}"
+  cancel-in-progress: true
+
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+
+jobs:
+  build_wheels:
+    if: "github.repository == 'MDAnalysis/mdanalysis'"
+    name: Build wheels
+    runs-on: ${{ matrix.buildplat[0] }}
+    strategy:
+      fail-fast: false
+      matrix:
+        buildplat:
+          - [ubuntu-18.04, manylinux_x86_64]
+          - [macos-10.15, macosx_*]
+          - [windows-2019, win_amd64]
+        python: ["cp38", "cp39", "cp310"]
+    defaults:
+      run:
+        working-directory: ./package
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.6.0
+        with:
+          package-dir: package
+        env:
+          CIBW_BUILD: ${{ matrix.python }}-${{ matrix.buildplat[1] }}
+
+      - name: upload artifacts
+        if: |
+          (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/package')) ||
+          (github.event_name == 'release' && github.event.action == 'published')
+        uses: actions/upload-artifact@v2
+        with:
+          path: wheelhouse/*.whl
+          retention-days: 7
+
+  build_sdist:
+    if: "github.repository == 'MDAnalysis/mdanalysis'"
+    name: build package source distribution
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./package
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Build sdist
+        run: pipx run build --sdist
```

Comment:
is `pipx` installed in ubuntu-latest?

Reply:
Yup, https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#package-management

Actually was quite surprising to find, I thought I would have needed to install it, but I guess not.